src/Evolution/DiscontinuousGalerkin/InitializeElement.hpp

```diff
@@ -114,8 +116,11 @@ struct InitializeElement {
   };
 
   // Tags related only to the system
+  template <typename System, typename = std::nullptr_t>
+  struct SystemTags;
+
   template <typename System>
-  struct SystemTags {
+  struct SystemTags<System, Requires<not System::is_conservative>> {
```

Comment:
Use  a `bool` as the second template parameter would be faster than SFINAE. You should be able to default it to `System::is_conservative`